nautobot/docs/installation/nautobot.md

```diff
@@ -49,46 +52,51 @@ Now that we've created the virtualenv, the remaining steps will be performed as
 !!! warning
     Don't skip this step!!
 
-    It's important to install Nautobot as the `nautobot` user so that we don't have to worry about fixing permissions later.
+    It is critical to install Nautobot as the `nautobot` user so that we don't have to worry about fixing permissions later.
 
 ```no-highlight
 $ sudo -iu nautobot
 ```
 
-After becoming `nautobot`, we need to set the `NAUTOBOT_ROOT` environment variable again for this user (since it is a fresh session as far as the system is concerned):
+### Update the Nautobot `.bashrc`
 
-```no-highlight
-$ export NAUTOBOT_ROOT=/opt/nautobot
-```
+After becoming `nautobot`, we need to set the `NAUTOBOT_ROOT` environment variable again for this user (since it is a fresh session as far as the system is concerned). 
 
-### Update the Nautobot `.bashrc`
+We want to make sure that this variable is always set.
 
-Add this to `~/.bashrc` for `nautobot` so that from here on out, anytime you become `nautobot`, your `NAUTOBOT_ROOT` will be set automatically.
+Add this to `~/.bashrc` for `nautobot` so that anytime you become `nautobot`, your `NAUTOBOT_ROOT` will be set automatically.
 
 ```no-highlight
 $ echo "export NAUTOBOT_ROOT=/opt/nautobot" >> ~/.bashrc
 ```
 
-## Activate the Virtual Environment
-
-!!! warning
-    This and all remaining steps in this document should all be performed as the `nautobot` user!
+Next, reload the `.bashrc` file so that your `NAUTOBOT_ROOT` is set and display its value to verify it worked:
 
-    Hint: Use `sudo -iu nautobot`
+```no-highlight
+$ source ~/.bashrc
+$ echo $NAUTOBOT_ROOT
+/opt/nautobot
+```
 
-To work inside a Python virtualenv, it must be activated. This makes sure that the version of Python you're using, as well any dependencies that you install remain isolated in this environment.
+## Understanding the Virtual Environment
 
-If it helps, try to think of activating the virtualenv like entering a shell unique to Nautobot's application environment.
+Because the `nautobot` user was created with `NAUTOBOT_ROOT` set as its home directory and we had you set the shell to `/bin/bash`, the binary path `$NAUTOBOT_ROOT/bin` is automatically added to the beginning of the `$PATH` environment variable:
 
 ```no-highlight
-$ source /opt/nautobot/bin/activate
-(nautobot) $
+$ echo $PATH
+/opt/nautobot/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin
 ```
 
-Observe that after activating, your prompt will now be preceded with the name of the virtualenv (`nautobot`).
+Therefore, any commands executed by the `nautobot` user will always check `$NAUTOBOT_ROOT/bin` first. 
+
+Because `NAUTOBOT_ROOT` also contains the Python virtualenv for Nautobot, all of the commands you will execute as the `nautobot` user, will automatically be used because they come first in the `$PATH`. 
```

Comment:
```suggestion
Because `NAUTOBOT_ROOT` also contains the Python virtualenv for Nautobot, all of the commands you will execute as the `nautobot` user will automatically prefer the virtualenv's commands because they come first in the `$PATH`. 
```